src/box/memtx_tx.c

```diff
@@ -3625,17 +3625,43 @@ memtx_tx_snapshot_cleaner_create(struct memtx_tx_snapshot_cleaner *cleaner,
 		if (clean == tuple)
```

Comment:
Unrelated to your patch, but while we are here. Is it ok that we clarify tuples to `prepared` ones, not `committed`?

Reply:
Can't say for sure, it's been this way ever since the MVCC snapshots are introduced.

Reply:
It's OK actually - we firstly freeze all prepared tuples, then we do `wal_sync` and `txn_limbo_confirm` - in other words, wait while all prepared tuples will become committed. So we must clarify prepared tuples here.